c7n/resources/cw.py

```diff
@@ -89,6 +90,30 @@ def process(self, resources):
                 AlarmNames=[r['AlarmName'] for r in resource_set])
 
 
+@Alarm.filter_registry.register('exclude-composite-alarms')
```

Comment:
Name of the filter could be different I think 😄 . These are all Metric Alarms

Reply:
`exclude-composite-child-alarms`?